### PR TITLE
VSCode-specific iteration on the `ClientNotFoundModal`

### PR DESCRIPTION
**Pull request by phryneas**
image
image
image
image

There's sure some space for visual improvements but this is a good first pass.

### CODE REVIEW

**Comment by apollo-bot2** [comment]
## Detected SAST Vulnerabilities
## :red_circle: Potential Security Issues Found

If you are seeing this message, it means that [the security scanning tool](https://semgrep.dev/) that Apollo uses to test our source has identified a potential security issue in code introduced or updated via your branch. Information about what was detected and steps to move forward are below.

If the scanner detected a legitimate issue, please take action to correct it prior to merging this PR. The action required will vary based on the detection. If the detection is a false positive, please follow the steps below to resolve the issue.

###  Issues Detected 
| Message | Caused CI Failure? | Link | Path | Signature |
|---------|--------------------|------|------|-----------|
| Insecure WebSocket Detected. WebSocket Secure (wss) should be used for all WebSocket connections. | Yes | <a href="https://github.com/apollographql/apollo-client-devtools/blob/1d319740b77eeafee630d1e51530c779027914fe/src/application/components/ClientNotFoundModal.vscode.tsx?plain=1#L59" target="_blank">Link</a> | src/application/components/ClientNo  tFoundModal.vscode.tsx | rules.providers.semgrep.security.ja  vascript.lang.security.detect-  insecure-websocket |
| To ensure reproducible and deterministic builds, use `npm ci` rather than `npm install` in scripts. This will use the lockfile rather than updating it. | No | <a href="https://github.com/apollographql/apollo-client-devtools/blob/1d319740b77eeafee630d1e51530c779027914fe/src/application/components/ClientNotFoundModal.vscode.tsx?plain=1#L37" target="_blank">Link</a> | src/application/components/ClientNo  tFoundModal.vscode.tsx | rules.providers.semgrep.security.ge  neric.ci.security.use-frozen-  lockfile-npm |
<hr />
<details>
    <summary><h3>False Positive Resolution Process</h3></summary>

The easiest way to resolve a false positive is to add a comment containing `nosemgrep` to the code that triggered the detection. You can add the comment directly to the line triggering the detection or on the line immediately above it. For example:

```python
#!/usr/bin/env python3

def my_function():
    my_code_generating_detection() # nosemgrep
```

<details>
    <summary><h4>What if I can't add a line comment?</h4></summary>

If you can't add a comment because the detection is generated by a file that does not support comments (like JSON), you can use Semgrep's `.semgrepignore` file. For more information about how to create an exclusion via `.semgrepignore`, refer to the [Semgrep documentation](https://semgrep.dev/docs/ignoring-files-folders-code/#defining-ignored-files-and-folders-in-semgrepignore).
</details>

</details>
<hr />
<details> 
    <summary><h3>Getting Help</h3></summary>


The Apollo Security team is available to assist in resolving this issue. Please tag us on this PR using  `@apollographql/security` if you need assistance! 
</details>
<hr />
<details>
    <summary><h3>How do I know I fixed this correctly?</h3></summary>
Once you have resolved the issue completely, this message will disappear! If you're still seeing this message, there is more to do prior to merging.
</details>

File: webpack.config.js [hunk 1]
```diff
@@ -101,7 +101,15 @@ export default /** @returns {import("webpack").Configuration} */ (env) => {
       chunkFormat: "commonjs",
     },
     resolve: {
-      extensions: [".mjs", ".js", ".ts", ".tsx", ".css"],
+      extensions: [
+        `.${target}.ts`,
```
Comment (jerelmiller) [inline webpack.config.js:105]: 🤯 I didnt realize it was this simple. Awwesome!

**Review by jerelmiller — APPROVED** [review]
I've added some updates to change some styling and reword a few sections. See if you like them!

Once you feel good about the changes, go ahead and merge. Let's see if we can get a first release of this out.

**Comment by jerelmiller** [comment]
Oh and don't forget to add a changeset!

**Pull request merged**